src/lib/holocene/icon-button.svelte

```diff
@@ -18,15 +18,11 @@
   export let label: string;
 </script>
 
-<button
-  type="button"
-  class={merge(
-    'inline-flex h-9 w-9 shrink-0 items-center justify-center rounded-full border-2 border-[transparent] text-primary outline-none hover:surface-interactive-secondary focus-visible:surface-interactive-secondary focus-visible:border-indigo-600 disabled:cursor-not-allowed disabled:opacity-50',
-    className,
-  )}
-  on:click|stopPropagation
+<Button
+  variant="ghost"
+  leadingIcon={icon}
+  class={merge('h-9 w-9 shrink-0 rounded-full p-0', className)}
```

Comment:
I don't love this, but `<Button />` already has so many different variations I'm not sure I love adding an `icon` or `round` variant.

Reply:
Yeah I just did this to avoid duplicating all the `ghost` variant `Button` styles here, but open to suggestions.

Reply:
Let's leave it as is, more just wanted to make a mental note that it may be something we want in the future.